UdpSockets.m

```diff
@@ -82,6 +82,20 @@ @implementation UdpSockets
     if (callback) callback(@[]);
 }
 
+RCT_EXPORT_METHOD(setBroadcast:(NSString*)cId
+                  flag:(BOOL)flag
+                  callback:(RCTResponseSenderBlock)callback) {
+    UdpSocketClient* client = [self findClient:cId callback:callback];
+    if (!client) return;
+
+    NSError *error = nil;
+    if (![client setBroadcast:flag error:&error])
+    {
+        callback(@[error]);
```

Comment:
should return here to avoid calling callback twice. 

Are you getting the same error I am?

<img width="373" alt="screen shot 2015-07-29 at 1 49 27 pm" src="https://cloud.githubusercontent.com/assets/83948/8965227/15d6d9a2-35f8-11e5-95c6-50b5dd97d860.png">